lib/msf/core/exploit/socket_server.rb

```diff
@@ -18,7 +18,7 @@ def initialize(info = {})
 
     register_options(
       [
-        OptAddress.new('SRVHOST', [ true, "The local host to listen on. This must be an address on the local machine or 0.0.0.0", '0.0.0.0' ]),
+        OptAddressLocal.new('SRVHOST', [ true, "The local host to listen on. This must be an address on the local machine or 0.0.0.0", '0.0.0.0' ]),
```

Comment:
```suggestion
        OptAddressLocal.new('SRVHOST', [ true, 'The local host or network interface to listen on. This must be an address on the local machine or 0.0.0.0.', '0.0.0.0' ]),
```

Note that this change will permeate to most modules. :+1:

Reply:
That missing period was bugging me.

Reply:
Punctuation-wise it's better but...

Reply:
Fair point, probably why it was done. You can revert this particular change if you'd like. That said...

```
msf5 exploit(windows/smb/smb_delivery) > help help
There's only so much I can do
msf5 exploit(windows/smb/smb_delivery) >
```

Reply:
Oh I didn't see I accepted it... it's ok :)

Reply:
See follow-up commit suggestion!